css/style.css

```
@@ -0,0 +1,353 @@
+  height: auto;
+}
+
+.main-navigation {
+  background-color: var(--basic-grey);
+}
+
+.navigation {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  text-decoration: none;
+  list-style: none;
+}
+
+.color-nav {
+  color: var(--basic-white);
+}
+
+.main-navigation a,
+.catalog-list a,
+.list-orders a {
+  color: var(--basic-white);
+}
+
+/* header */
+.search {
+  Font family: Cuprum;
+  Font style: Regular;
+  Font size: 18px;
+  Line height: 24px;
+  background-color: var(--focus-nav);
+}
+
+.list-orders {
+  Font family: Cuprum;
+  Font style: Regular;
+  Font size: 18px;
+  Line height: 24px;
+  color: var(--basic-white);
+  background-color: var(--focus-nav);
+  list-style: none;
+}
+
+.ordering {
+  background-color: var(--basic-green);
+}
+
+.title {
+  font-family: Cuprum;
+  font-style: italic;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 23px;
+  color: var(--basic-red);
+}
+
+.telephone {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 21px;
+  line-height: 30px;
+  color: var(--basic-red);
+  text-align: center;
+}
+
+.address {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 24px;
+  color: var(--address);
+  text-align: center;
+}
+
+.user-navigation {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 21px;
+  line-height: 21px;
+  background-color: var(--basic-white);
+  text-align: right;
+}
+
+.user-navigation a {
+  color: var(--basic-black);
+}
+
+/*main*/
+
+
+.catalog-item h2{
+  font-family: Cuprum;
+font-style: normal;
+font-weight: bold;
+font-size: 24px;
+line-height: 30px;
+color: var(--basic-white)
+
+}
+.material {
+  background-color: var(--special-ylellow);
+}
+
+.tools {
+  background-color: var(--basic-blue);
+}
+
+.tech {
+  background-color: var(--basic-purple);
+}
+
+.sales {
+  background-color: var(--green);
+}
+
+.deliver {
+  background-color: var(--ylellow);
+}
+.catalog-slide{
+  background-image: url("../img/true-man.jpg");
+   background-repeat: no-repeat;
+}
+
+.popular-staff {
+  font-family: Cuprum;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 30px;
+  line-height: 30px;
+  color: var(--basic-black);
+  background-color: var(--special-gray);
+}
+
+.drill-title {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  color: var(--basic-black)
+}
+
+.old-price {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 18px;
+  color: var(--old-price);
+}
+
+.buy-button {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 18px;
+  color: var(--basic-white);
+  background-color: var(--basic-red);
+}
+
+.firms-list,
+.drill-list {
+  list-style: none;
+}
+
+.popular-firms-section,
+.popular-product-section {
```

```suggestion
.popular-firms,
.popular-product {
```
Лучше не добавлять section